src/Classes/Validator.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Emanate\BeemSms\Classes;
+
+use Emanate\BeemSms\Exceptions\InvalidPhoneAddress;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Str;
+
+final class Validator
+{
+    /**
+     * List of phone address prefixes.
+     *
+     * @var array<string>
+     */
+    protected static array $phoneAddressPrefix = [
```

Comment:
@LogicSatinn can we move this to config file ili iwe dynamic for users?? ... let's say for outside tz

Reply:
@JemCdo Do Beem send messages to outside TZ ??

I don't think they do.

Reply:
They do nini?

Reply:
They do ... angalia coverage yao on their website

Reply:
They actually do send messages internationally.... but I think they use a different set of APIs.

Reply:
not sure, I will merge this for now